modules/apim_common/manifests/params.pp

```diff
@@ -41,46 +45,54 @@
   case $profile {
     'apim_analytics_dashboard': {
       $pack = "wso2am-analytics-${version}"
+      # $remote_pack = "http://10.100.5.174:8000/packs/wso2am-analytics-2.6.0.zip"
```

Comment:
Shall we remove the IP address and add a placeholder? WDYT?

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/128/commits/775a688e3eae96b3f6c013f85e1fed7bd19ae827